src/V3Sched.cpp

```diff
@@ -451,14 +452,12 @@ void orderSequentially(AstCFunc* funcp, const LogicByScope& lbs) {
                         if (VN_IS(procp, Always)) {
                             subFuncp->slow(false);
                             FileLine* const flp = procp->fileline();
-                            bodyp = new AstWhile{
-                                flp,
-                                // If we change to use exceptions to handle finish/stop,
-                                // this can get removed
-                                new AstCExpr{flp,
-                                             "VL_LIKELY(!vlSymsp->_vm_contextp__->gotFinish())", 1,
-                                             true},
-                                bodyp};
+                            AstNodeExpr* const condp = new AstCExpr{
+                                flp, "VL_LIKELY(!vlSymsp->_vm_contextp__->gotFinish())", 1, true};
```

Comment:
Really gotFinish should be in all loops unless we know they have a terminating condition. Mostly noting this as that's a fix/cleanup unrelated to this pull.